app/src/docs/components/ApiPage/Content.mdx

```diff
@@ -25,5 +25,17 @@ If you'd like to contribute a client library and get it listed here, please get
 ## Authentication
 A session token is required to make requests to the REST API endpoints or over the websocket protocol. You can obtain a session token by authenticating either using an API key or by signing a cryptographic challenge using an Ethereum private key.
 
-Once you get a session token using one of the below methods, see the section on using it.
-</div></ScrollableAnchor>
\ No newline at end of file
+Once you get a <kbd>session token</kbd> using one of the below methods, see the section on using it.
+
+### Response code
+
+code | Description
```

Comment:
Capital `C` for `code`?

Reply:
Well, I'd say so too :P Designs show lowercase but maybe I'll just add it anyway 🤫
 
https://share.goabstract.com/3fe2aa03-3291-425f-ad69-4fb0e191a670